django_loose_fk/filters.py

```diff
@@ -53,6 +57,9 @@ def filter(self, qs, value):
 
         if local:
             local_object = get_resource_for_path(parsed.path)
+            if self.instance_path:
+                for bit in self.instance_path.split("."):
```

Comment:
Seems a bit naive to make it go one level at max using split? Leaving for Sergei to judge.

Reply:
but it's not? `split` returns a list of attributes, and each attribute is accessed

Reply:
Hmm brain fluke of mine ...